app/models/measure_type.rb

```diff
@@ -24,11 +24,20 @@ class MeasureType < Sequel::Model
     '696', # Additional duties (safeguard)
   ].freeze
 
-  UNIT_EXPRESSABLE_MEASURES = [
-    'C', # Applicable duty
-    'D', # Anti-dumping/countervailing measures
-    'J', # Countervailing charge
-  ].freeze
+  if Rails.application.config.include_excise_measure_units
+    UNIT_EXPRESSABLE_MEASURES = [
+      'C', # Applicable duty
+      'D', # Anti-dumping/countervailing measures
+      'J', # Countervailing charge
+      'Q', # Excise
+    ].freeze
+  else
+    UNIT_EXPRESSABLE_MEASURES = [
+      'C', # Applicable duty
+      'D', # Anti-dumping/countervailing measures
+      'J', # Countervailing charge
+    ].freeze
```

Comment:
Not too precious about it but how would you feel adding 
```ruby
      'C', # Applicable duty
      'D', # Anti-dumping/countervailing measures
      'J', # Countervailing charge
```
to a constant and then you do something like:
```ruby
if Rails.application.config.include_excise_measure_units
    UNIT_EXPRESSABLE_MEASURES =  UNIT_EXPRESSABLE_MEASURES + [
      'Q', # Excise
    ].freeze
  else
    UNIT_EXPRESSABLE_MEASURES 
```
?